bip-XXXX-Bech32_Encoded_Transaction_Postion_References.mediawiki

```diff
@@ -0,0 +1,314 @@
+
+Bitcoin uses regularly use TxID (Transaction Identifiers) for referencing individual transactions. TxID's are a core part of the consensus protocol and were naturally exported to the user-space since the first release of Bitcoin.
+
+However, for many use-cases they have practical limitations:
+* TxID's are expensive for full nodes to lookup.
+* TxID's require third-party service for SPV wallets to lookup.
+* TxID's are very long HEX encoded values (64 characters long).
+* TxID's are without checksums.
+For transactions that have been embedded in the blockchain, it is possible to reference them not by their ID, but by their location within the blockchain itself. In this document, we propose a standard for doing this.
+
+=== Examples ===
+All examples use the ''Bitcoin Display Format 0'' described in ''Bitcoin Display Formats'' appendix.
+* Genesis Coinbase Transaction: <tt>tx1-rqqq-qqqq-qmhu-qk</tt>
+* Transaction #2205 of Block #466793: <tt>tx1-rjk0-u5ng-4jsf-mc</tt>
+
+== Specification ==
+
+A '''confirmed transaction position reference''', or '''TxRef''' is reference to a particular location within the blockchain specified by the block height and the transaction position.
+
+''Please Note: All values in this specification are encoded in little-endian format.''
+
+A TxRef may reference a location that doesn't exist because:
+* The block height hasn't been minded yet.
+* The Transaction Position is more than the total transaction within the referenced block.
+Therefore, implementers must be careful not to display TxRef's to users prematurely:
+* Applications MUST NOT display TxRef's for transactions with less than 6 confirmations.
+* Application MUST show a warning for TxRef's for transactions with less than 100 confirmations.
+** This warning SHOULD state that in the case of a large reorganization, the TxRefs Displayed may point to a different transaction, or to no transaction at all.
+
+=== Human Display Formats ===
+This specification provides standardisation of encoding and namespacing for '''application display formats'''.
+
+If a display format doesn't comply with this standard, it MUST be called a '''non-standard application display format'''.
+
+==== Encoding ====
+* All display formats MUST be encoded with standard Bech32<ref>'''Why use Bech32 Encoding for Confirmed Transaction References?''' The error detection and correction properties of this encoding format make it very attractive. We expect that it will be reasonable for software to correct a maximum of two characters; however, we haven’t specified this yet.</ref> encoding as defined within the [https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki BIP 0173] specification.
+* The Human Readable Part of the Bech32 encoding MUST be "tx"<ref>'''Why use the same Human Readable Part for different systems or blockchains?''' The ‘tx1’ prefix is to namespace the Bech32 encoding to this specification. We use Magic Values internally to namespace between projects.</ref>.
```

Comment:
The human readable part is to inform users that the string should conform to this particular BIP.  We have out own magic and versioning system so that we can upgrade and extend the format transparently for our users.